generator/input/new_description.json

```diff
@@ -29,17 +29,17 @@
       "processes": 2,
       "endpoints": [
         {
-          "name": "/end-1",
+          "name": "end1",
           "protocol": "http",
           "cpu_consumption": 0.003,
           "network_consumption": 0.002,
           "memory_consumption": 0.003,
           "forward_requests": "asynchronous",
           "called_services": [
             {
-              "service": "service-2",
+              "service": "service2",
               "port": "80",
-              "endpoint": "/end-2",
+              "endpoint": "end2",
```

Comment:
Is there any restriction with special symbols? I guess that's why you had to change their format...

Reply:
Yes, for Now I'd suggest to don't use any specific symbols like dashes or underline. We can add the support of these symbols in #25.